ush/setup.py

```diff
@@ -762,6 +771,21 @@ def get_location(xcs, fmt, expt_cfg):
             else:
                 grid_config[param] = value
 
+    # This logic belongs in predef_grid_params.yaml once merged with make_grid integration.
+    if predef_grid_name == "RRFS_NA_3km":
+        fv3_namelist = expt_config["task_run_fcst"]["fv3"]["namelist"]
+        fv3_namlelist["update_values"]["fms2_io_nml"]["netcdf_default_format"] = "netcdf4"
+
+    # Load model write component grid settings
+    quilting_cfg = get_yaml_config(Path(USHdir, "quilting.yaml"))
+    if not quilting:
+        update_dict(quilting_cfg["no_quilting"], expt_config)
+    else:
+        write_grid = expt_config["task_run_fcst"]["WRTCMP_output_grid"]
+        update_dict(quilting_cfg[write_grid], expt_config)
```

Comment:
This logic replaces what was originally in the Jinja template for model_configure.